### PR TITLE
added summary breaks to all the adoc files using the more tag

### PR DESCRIPTION
**Pull request by mwitzenm**
#### What is this PR About?
I modified the adoc files to include a more tag to the documents so the summary page could be shorter and not just the first 70 lines of the file

#### How should we test or review this PR?
Spin up your local hugo and test my pr

#### Is there a relevant Trello card or Github issue open for this?
https://github.com/redhat-cop/uncontained.io/issues/41

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----

- [x] Have you followed the [contributing guidelines](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the uncontained.io guides?

-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: site/content/articles/ldap-group-sync.adoc [hunk 1]
```diff
@@ -17,10 +17,12 @@ v1.0, 2017-06-10
 :toc-title:
 
 include::site/layouts/variables.adoc[]
-toc::[]
 
 NOTE: This guide discusses the synchronization of groups defined in an LDAP server with OpenShift and is distinct from using an LDAP server to authenticate users to OpenShift. Please refer to the link:/playbooks/installation/ldap_integration{outfilesuffix}[LDAP Integration guide] for using an LDAP server as an identity provider to govern user authentication to OpenShift.
```
Comment (etsauer) [inline site/content/articles/ldap-group-sync.adoc]: Let's remove the`NOTE:` from this intro text

File: site/content/guides/openshift-admin-overview.adoc [hunk 2]
```diff
@@ -16,11 +16,13 @@ v1.0, 2016-05-17
 :toc: macro
 :toc-title:
 include::site/layouts/variables.adoc[]
-toc::[]
 
 == Overview
 
 This is a brief overview to aimed at system administrators aimed at getting accustomed to what OpenShift actually looks like installed on a host, and common places to look when things go wrong.
```
Comment (etsauer) [inline site/content/guides/openshift-admin-overview.adoc]: `s/overview to aimed/overview aimed/`

File: site/content/guides/openshift-load-balancing.adoc [hunk 3]
```diff
@@ -11,386 +11,395 @@ tags:
...
-
-+
-We will need to configure AWS ELB to persist HTTPS sessions via the "ssn" cookie.
-See SessionName under master-config.yaml
-  https://docs.openshift.com/enterprise/3.2/install_config/configuring_authentication.html
-
-+
-Follow the "Application-Controlled Session Stickiness" procedure in http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-sticky-sessions.html to set your ELB to be aware of the OpenShift session.
+
+Interating with a Load Balancing scheme to ensure high availability is mantained to the cluster components
```
Comment (etsauer) [inline site/content/guides/openshift-load-balancing.adoc]: This statement has a few spelling issues, and doesn't fully make sense.

File: site/content/guides/openshift-load-balancing.adoc [hunk 4]
```diff
@@ -20,7 +20,7 @@ v1.0, 2015-10-30
 
 include::site/layouts/variables.adoc[]
 
-Interating with a Load Balancing scheme to ensure high availability is mantained to the cluster components 
+This guide will walk you how to leverage Load Balancers for cluster and application high availability.
```
Comment (etsauer) [inline site/content/guides/openshift-load-balancing.adoc]: `walk you THROUGH`

Comment (mwitzenm) [reply site/content/guides/openshift-load-balancing.adoc]: should be fixed already, I see it in the file.

**Pull request merged**